### PR TITLE
Dto class added

### PR DESCRIPTION
**Pull request by raviraj-12**

### CODE REVIEW

File: DTo/guii.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hii tehre is dto class'
```
Comment (ravirajgundgole) [inline DTo/guii.txt:1]: please add print statement

File: DTo/guii.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 hii tehre is dto class'
+
```
Comment (ravirajgundgole) [inline DTo/guii.txt:2]: remove this extra line

Comment (raviraj-12) [reply DTo/guii.txt:2]: removed extra line

Comment (ravirajgundgole) [reply DTo/guii.txt:2]: did not remove yet

Comment (raviraj-12) [reply DTo/guii.txt:2]: i will remove tomorrow

**Review by ravirajgundgole — APPROVED** [review]

**Pull request merged**